kolibri/core/templates/kolibri/base.html

```diff
@@ -1,10 +1,8 @@
-{% load i18n kolibri_tags %}
-<html>
+{% load i18n kolibri_tags webpack_tags %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
-  {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
-  {% base_frontend_async %}
-  {% base_frontend_sync %}
+  {% webpack_assets 'default_frontend' %}
+  {% base_frontend_assets %}
```

Comment:
As I understood the previous intention by Richard, we wanted to be able to either register an asset to be: 

 * auto-loaded together with all the other front end assets. That happens in `{% base_frontend_assets %}`
 * To be loaded on-demand... that's the `{% webpack_assets 'default_frontend' %}` example
 * To be loaded by event identifiers... that's the `async` stuff. But I found the names containing `sync` to be a bit confusing for my own understanding. As perhaps an old-school frontend dev I was just thinking "how do I include an asset".. the sync/async was a bit over my head, so I went for a more webpack-oriented definition. Like, "using this tag means using webpack for building and including your JS asset. Done".

But maybe there are better naming conventions?